staging/src/k8s.io/api/storage/v1/types.go

```diff
@@ -57,17 +58,17 @@ type StorageClass struct {
 	// +optional
 	MountOptions []string `json:"mountOptions,omitempty" protobuf:"bytes,5,opt,name=mountOptions"`
```

Comment:
something like 
"Dynamically provisioned PersistentVolumes of this storage class are created with these mountOptions" → "mountOptions controls the mountOptions for dynamically provisioned PersistentVolumes of this storage class"